Update communications.md (#20550)
fleetdm/fleet: handbook/company/communications.md

## Getting a contract signed

> If a contract is ready for signature and requires no review or revision, the requestor logins into DocuSign using the login from the 1Password vault and routes the agreement to the CEO for signature. 

Reviewer: ```suggestion
If a contract is ready for signature and requires no review or revision, log into DocuSign (credentials in 1Password) and route the agreement to the CEO for signature. 
```

When a contract is going to be routed for signature by someone outside of Fleet (i.e. the vendor or customer), the requestor is responsible for working with the other party to make sure the document gets routed to the CEO for signature.

The SLA for contract signature is **2 business days**. Please do not follow up on signature unless this time has elapsed.

_**Note:** Signature open time for the CEO is not currently measured, to avoid the overhead of creating separate signature issues to measure open and close time. This may change as signature volume increases._

> > _**Note:** If a contract is ready for signature and requires no review or revision, the requestor logins into DocuSign using the login from 1Password and routes the agreement to the CEO for signature._ 

Reviewer: ```suggestion
> _**Note:** If a contract is ready for signature and requires no review or revision, log into DocuSign (credentials in 1Password) and route the agreement to the CEO for signature._ 
```
